docs/how-to/tour.rst

```diff
@@ -0,0 +1,122 @@
+Tour of the Code
+================
+
+Before getting started, it's nice to know a bit about how the project is structured and where
+to look for examples. This document aims to provide a brief tour of the
+important features of the code. It's aimed at new contributors, but frequent flyers can
+skip down to the bottom for a quick reference.
+
+General Structure
+-----------------
+
+Core Code
+*********
+
+Batavia implements the Python native types, builtins, and standard library partially in JavaScript.
+This ensures quick execution and less compiling of the compiler. These are orgainzed into the
+``builtins``, ``core``, ``modules``, ``stdlib``, and ``types`` files of the main ``/batavia`` directory, with
+corresponding subdirectories. Alongside the virtual machine and test suite, this code makes 
+up the bulk of Batavia. New contributors should start with the ``types`` and ``builtins`` sections
+for the best examples to review and copy from. These implementations are the foundation of Python as you know it and
+should be immediately familiar to a Python developer.
+
+Support
+*******
+You'll also notice folders for tests, docs, and a few other bits and bobs, like ``testserver``, which is
+a Django project that allows code execution from your browser. Contributions to the 
```

Comment:
might be better to describe testserver as a sample deployment that lets you test Batavia.